pkg/operation/botanist/addons.go

```diff
@@ -434,6 +434,40 @@ func (b *Botanist) generateCoreAddonsChart(ctx context.Context) (*chartrenderer.
 				return nil, err
 			}
 		}
+	} else if b.Shoot.ReversedVPNEnabled {
+		var (
+			vpnTLSAuthSecret = b.Secrets["vpn-seed-server-tlsauth"]
+			vpnShootSecret   = b.Secrets["vpn-shoot-client"]
+			vpnShootConfig   = map[string]interface{}{
+				"endpoint":       b.outOfClusterAPIServerFQDN(),
+				"port":           "8132",
+				"podNetwork":     b.Shoot.Networks.Pods.String(),
+				"serviceNetwork": b.Shoot.Networks.Services.String(),
+				"tlsAuth":        vpnTLSAuthSecret.Data["vpn.tlsauth"],
+				"vpnShootSecretData": map[string]interface{}{
+					"ca":     vpnShootSecret.Data["ca.crt"],
+					"tlsCrt": vpnShootSecret.Data["tls.crt"],
+					"tlsKey": vpnShootSecret.Data["tls.key"],
+				},
+				"reversedVPN": map[string]interface{}{
+					"enabled": true,
+				},
+				"podAnnotations": map[string]interface{}{
+					"checksum/secret-vpn-shoot-client": b.CheckSums["vpn-shoot-client"],
+				},
+			}
+		)
```

Comment:
Use `struct` with `json` tags instead of `map[string]interface{}`

Reply:
What for? We plan to refactor this as part of #2754 anyways entirely and plan to get rid of the Helm chart, so let's not invest too much into this now and rather do it similar to how the other charts are currently handled/rendered.

Reply:
Okay